docs/config.md

```diff
@@ -282,7 +282,7 @@ Instead of tagging, you also have the option to add metadata to instances. This
    ```
 
 ## Timeout settings
-During some heavy workload cloud, the time for create and delete openstack instance might takes long time, by default it's 5 minute.
-you can set:
-`CLUSTER_API_OPENSTACK_INSTANCE_DELETE_TIMEOUT` for instance delete timeout value.
-`CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT` for instance create timeout value.
+During some heavy workload cloud, the time to create and delete openstack instance might take long time, by default it's 5 minute.
+You can set:
+`TimeoutInstanceDelete` for instance delete timeout value and
+`TimeoutInstanceCreate` for instance create timeout value in file `pkg/cloud/openstack/machine/actuator.go`
```

Comment:
others looks good, except htis one are you saying we need change the code in order to get the time out value? 
I think originally the code is trying to use environment variable so no code change @prankul88

Reply:
Ohh okay. Will revert this change. Thanks

Reply:
Hi @jichenjc 
Kindly check, I have updated the patch.